src/pages/test-results-search/components/search-result/search-result.ts

```diff
@@ -41,6 +42,27 @@ export class SearchResultComponent {
     return `${name.title} ${name.firstName} ${name.lastName}`;
   }
 
+  driverRider(searchResult: SearchResultTestSchema): string {
```

Comment:
is there not already a helper for this, which you can just pass the category code to? 
src/shared/helpers/driver-type.ts